skus/controllers_v2.go

```diff
@@ -0,0 +1,154 @@
+package skus
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/brave-intl/bat-go/middleware"
+	"github.com/brave-intl/bat-go/utils/handlers"
+	"github.com/brave-intl/bat-go/utils/inputs"
+	"github.com/brave-intl/bat-go/utils/requestutils"
+	"github.com/go-chi/chi"
+	uuid "github.com/satori/go.uuid"
+)
+
+// RouterV2 for order endpoints v2
+func RouterV2(service *Service) chi.Router {
+	r := chi.NewRouter()
+
+	r.Route("/{orderID}/credentials", func(cr chi.Router) {
+		cr.Use(corsMiddleware([]string{"GET", "POST"}))
+		cr.Method("POST", "/", middleware.InstrumentHandler(
+			"CreateOrderCredsV2", CreateOrderCredsV2(service)))
+		cr.Method("GET", "/", middleware.InstrumentHandler(
+			"GetOrderCredsV2", GetOrderCredsV2(service)))
+		cr.Method("GET", "/{itemID}", middleware.InstrumentHandler(
+			"GetOrderCredsByIDV2", GetOrderCredsByIDV2(service)))
+	})
+
+	return r
+}
+
+// CreateOrderCredsV2Request includes the item ID and blinded credentials which to be signed
+type CreateOrderCredsV2Request struct {
+	ItemID       uuid.UUID `json:"itemId" valid:"uuid"`
+	BlindedCreds []string  `json:"blindedCreds" valid:"base64"`
+}
+
+// CreateOrderCredsV2 is the handler for creating order credentials
+func CreateOrderCredsV2(service *Service) handlers.AppHandler {
```

Comment:
@husobee 

Doesn't this need to return an `order_id` or an order in the response to mimic the flow in `skus_e2e_works` https://github.com/brave/brave-core/blob/master/components/skus/browser/rs/lib/tests/e2e.rs#L102 for `refresh`, `fetch`, and `present`?

Reply:
Ah, create order is different than create order creds, in that the caller will send blinded tokens in the create order creds request to have them signed by challenge bypass.  For our purposes one can use the existing v1 version of the create order as it is, there is no change to that api.